openapi_python_client/templates/model.pyi

```diff
@@ -66,8 +83,37 @@ class {{ model.reference.class_name }}:
     {% endif %}
 
 {% endfor %}
-        return {{ model.reference.class_name }}(
+        {{model.reference.module_name}} = {{ model.reference.class_name }}(
 {% for property in model.required_properties + model.optional_properties %}
             {{ property.python_name }}={{ property.python_name }},
 {% endfor %}
         )
+
+        {% if model.additional_properties %}
+        {{model.reference.module_name}}._additional_properties.update(d)
```

Comment:
Why update vs set directly? I would think assignment would be faster but maybe not.